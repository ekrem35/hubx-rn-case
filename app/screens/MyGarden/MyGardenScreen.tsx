import React from 'react';

import {View} from 'react-native';

export default function MyGarden() {
  return <View style={{flex: 1, backgroundColor: '#013cFa50'}} />;
}
