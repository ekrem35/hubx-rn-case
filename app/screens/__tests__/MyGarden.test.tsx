/**
 * @format
 */

import 'react-native';
import React from 'react';
import MyGardenScreen from '../MyGarden/MyGardenScreen';

import {it} from '@jest/globals';

import {render, cleanup} from '@testing-library/react-native';

afterEach(cleanup);

describe('test my garden screen', () => {
  it('renders correctly', async () => {
    const screen = await render(<MyGardenScreen />);

    expect(screen.toJSON()).toMatchSnapshot();
  });
});
