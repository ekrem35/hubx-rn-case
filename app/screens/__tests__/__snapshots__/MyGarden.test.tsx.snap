// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test my garden screen renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#013cFa50",
      "flex": 1,
    }
  }
/>
`;
